Description: Disable "unstable" feature
 That way, the autopkgtest won't fail on ppc64el & s390x arch.
 .
 As suggested by upstream at
 https://github.com/huonw/primal/issues/60#issuecomment-2161791804
Author: Fab Stz <fabstz-it@yahoo.fr>
Last-Update: 2024-09-22

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,4 +54,3 @@
 [features]
 safe = []
 slow_tests = []
-unstable = []
